src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.DeserializeAsyncEnumerable.cs

```diff
@@ -7,7 +7,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
-using Utf8MemoryStream = System.Text.Json.Tests.Serialization.CollectionTests.Utf8MemoryStream;
```

Comment:
FWIW the `System.Text.Json.Tests.Serialization` namespace has crept up in quite a few other places as well.